tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist =
     # list of supported Django/Python versions:
     # https://docs.djangoproject.com/en/3.1/faq/install/#what-python-version-can-i-use-with-django
-    py{36,37,38,39}-dj{22,30,31}
+    py{36,37,38,39}-dj{22,30,31,32}
     py{38,39}-djmain
```

Comment:
Let's add Django 4.0 support as well while we're here?

Reply:
Django 4.0 is not yet final so better to wait for the final release which is due in December.
https://code.djangoproject.com/wiki/Version4.0Roadmap

Reply:
It's final in terms of features and changes. The only patches being applied to the 4.0 branch are bug fixes. Folk are adding the classifier to their packages already. See:

https://pypi.org/search/?c=Framework+%3A%3A+Django+%3A%3A+4.0

Reply:
@smithdc1 I'm waiting for your review, thanks